### PR TITLE
ci-operator: allow configuring observers, serve their config

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

Part of [DPTP-1668](https://issues.redhat.com/browse/DPTP-1668)

### CODE REVIEW

File: pkg/api/types.go [hunk 1]
```diff
@@ -602,6 +638,8 @@ type LiteralTestStep struct {
 	// Cli is the (optional) name of the release from which the `oc` binary
 	// will be injected into this step.
 	Cli string `json:"cli,omitempty"`
+	// Observers are the observers that should be running
```
Comment (alvaroaleman) [inline pkg/api/types.go:641]: Nit: It would be nice to write one sentence about what an oberver is here

Comment (stevekuznetsov) [reply pkg/api/types.go:641]: I expect to add an architecture page to the docs site for this

**Pull request merged**